bcl2fastq/index_summary.py

```diff
@@ -44,18 +42,27 @@
                 -  multiple select gives error due to inconsistent values
           """
 layout = html.Div(children=[
+    dcc.ConfirmDialog(
+        id='error',
+        message='You have input an incorrect run. Click either "Ok" or "Cancel" to return to the most recent run.'
+    ),
     dcc.Dropdown(
         id='run_select',
-        options=all_runs,
-        value=all_runs[0]['value'],
+        #   Options and Value attributes are concantenated string versions of all_runs.
+        #   Value selects a single copy of the run name.
+        options=[{'label': r, 'value': r} for r in all_runs],
+        value=([{'label': r, 'value': r} for r in all_runs])[0]['value'],
```

Comment:
I think this can be simplified -- `all_runs` is a list of run names, and we want the first item in this list to be selected by default. `value` is the string value of the item to be selected. Would it work as `value=all_runs[0]`?

Reply:
Yes it works, updating commit